frontend/reactnative/App.js

```diff
@@ -2,6 +2,8 @@ import React, {Component} from 'react';
 import {Text, View, StyleSheet} from 'react-native';
 import {Location, Permissions, MapView} from 'expo';
 
+import * as Config from './lib/Config.js';
```

Comment:
not sure if import * is good coding standards we should possibly start a discussion on this